.gitignore

```diff
@@ -5,3 +5,4 @@
 .idea
 node_modules
 npm-debug.log
+bower_components
```

Comment:
Hmm.  We currently host the production version of this (350map.org) directly on Github Pages.  I think this gitignore means that the hosted production version won't have the necessary files, right?  I won't be able to build locally and check in the files, and (I think) Github Pages doesn't have any kind of build hooks system.  

Any thoughts / advice / best practices for handling this?  (Separating production // gh-pages from the master branch would probably be a good first step for me to take...)